coffee/directives/fixtable.coffee

```diff
@@ -229,9 +255,16 @@ angular.module 'fixtable'
 					scope.currentDragScope = null
 					scope.$apply()
 
+			if scope.options.pagingOptions
+				scope.$on scope.options.pagingOptions.reloadEvent, ->
+					getPageData(true)
+
 			updateData = ->
 				# run callback method to get sorted/filtered data
-				if scope.options._paging() then getPageData()
+				if scope.options._paging()
```

Comment:
Seems like we could just rip this options.paging/_paging option out in favor of checking against the presence of options.pagingOptions. On the other hand it's always been here, so we might as well leave it.

Reply:
I feel more comfortable leave it for now. pagingOptions is really for a different type of paging model due to back end limitations. This model is also why the hasNextPage flag exists now. The back end doesn't know how many pages there are in total until we are on the last page.